baselayout/coreos-profile.sh

```diff
@@ -2,8 +2,21 @@
 
 # Only print for interactive shells.
 if [[ $- == *i* ]]; then
-	if ! systemctl is-active locksmithd > /dev/null; then
+	if ! [ -f  /run/update-engine/coordinator.conf ]; then
+		# No coordinator running
 		echo -e "Update Strategy: \033[31mNo Reboots\033[39m"
+	elif flock --nonblock --exclusive /run/update-engine/coordinator.conf --command /bin/true; then
+		# Coordinator ran, but isn't currently running; something's broken
+		echo -e "Update Strategy: \033[31mNo Coordinator\033[39m"
+	else
+		# Coordinator held the exclusive lock as of a moment ago, let's trust it
+		(
+			# subshell to avoid env pollution
+			source /run/update-engine/coordinator.conf
+			if [ "${STATE}" != "running" ]; then
```

Comment:
It might be better to have a switch on `STATE` since some, besides running, might not warrant  a red scare-message